opba-db/src/main/resources/migration/migrations/0000-create-table-banks.xml

```diff
@@ -120,4 +120,10 @@
         </sql>
     </changeSet>
 
+    <changeSet id="set_preffered_approach_dkb" author="mhr" context="xs2a-banks">
+        <sql dbms="postgresql" stripComments="true">
+            UPDATE opb_bank_profile SET preferred_approach = 'EMBEDDED', try_to_use_preferred_approach = true WHERE adapter_id = 'dkb-adapter';
```

Comment:
Please introduce a special approach for DKB or special approach setting that indicates we should execute OAuth2-embedded pre-step

Reply:
I am not sure if a new setting is required as it is actually done base on the bank profile  preferred_approach  field and the PreAuthorisationException thrown by the adapter